scripts/test-runner.js

```diff
@@ -257,10 +306,18 @@ function runTest(name, code, options, args) {
           }
         }
         if (markersIssue) break;
+        let codeToRun = addedCode + newCode;
+        if (args.es5) {
+          codeToRun = transformWithBabel(codeToRun, [],[["env",{forceAllTransforms: true, modules:false}]])
+        }
         try {
-          actual = exec(addedCode + newCode);
+          if (execSpec)
+             actual = execExternal(execSpec, codeToRun);
+          else
+             actual = execInContext(codeToRun);
         } catch (e) {
-          actual = e;
+          // always compare strings.
+          actual = "" + e;
```

Comment:
Is e.toString() more readable?

Reply:
Its a pretty common JS idiom to force string conversion (and actually faster than toString !)